### DISCUSSION TITLE
how to remove the botton of related module?

### DISCUSSION THREAD

**Discussion by leo719044** [YetiForceCompany/YetiForceCRM#15516] [How to] [answered]
how to remove the botton of related module? Anyone can help me ?

![image](https://user-images.githubusercontent.com/88583995/144783020-4585dd97-b62f-4c1b-902c-b58672205e83.png)

**Comment by leo719044** [comment, marked as answer]
like remove or add "Select document" from Document modules.

**Reply by vovpff** [reply]
You can do it manually in database. Soory, but I don't remember table name

**Reply by leo719044** [reply]
thanks a lot, I found it in "vtiger_relatedlist" in database. remove Select from "action"
